lib/routes/podcast/boost_message_dialog.dart

```diff
@@ -46,50 +59,68 @@ class BoostMessageDialogState extends State<BoostMessageDialog> {
   }
 
   Widget _buildMessageWidget() {
-    return Form(
-      key: _formKey,
-      autovalidateMode: AutovalidateMode.disabled,
-      child: TextFormField(
-        autovalidateMode: AutovalidateMode.disabled,
-        focusNode: _messageFocusNode,
-        controller: _boostMessageController,
-        keyboardType: TextInputType.multiline,
-        textInputAction: TextInputAction.done,
-        maxLines: null,
-        maxLength: 90,
-        maxLengthEnforcement: MaxLengthEnforcement.enforced,
-        validator: (raw) {
-          if (raw.length == 0) {
-            return "Please enter a message";
-          }
-          return null;
-        },
-        style: Theme.of(context)
-            .dialogTheme
-            .contentTextStyle
-            .copyWith(height: 1.0),
-      ),
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Form(
+          key: _messageKey,
+          autovalidateMode: AutovalidateMode.disabled,
+          child: TextFormField(
+            autovalidateMode: AutovalidateMode.disabled,
+            focusNode: _messageFocusNode,
+            controller: _messageController,
+            keyboardType: TextInputType.multiline,
+            textInputAction: TextInputAction.next,
+            maxLines: null,
+            maxLength: 90,
+            maxLengthEnforcement: MaxLengthEnforcement.enforced,
+            style: Theme.of(context)
+                .dialogTheme
+                .contentTextStyle
+                .copyWith(height: 1.0),
+            decoration: InputDecoration(
+              labelText: "Boostagram (optional)",
+            ),
+          ),
+        ),
+        CustomAmountForm(
```

Comment:
Extracting out the CustomAmount to its own widget is a good decision. I think though it is better to make it a `FormField` instead of a form. This way it can be inlined inside a top level form as we have here ( same pattern as with `AmountFormField`).